packages/core-ethereum/src/index.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { dbMock, debug, stringToU8a } from '@hoprnet/hopr-utils'
+import sinon from 'sinon'
+import HoprEthereum, { ChainWrapperSingleton, Indexer, useFixtures } from '.'
+import { sampleChainOptions } from './ethereum.mock'
+import { ACCOUNT_A, PARTY_A } from './fixtures'
+
+const namespace = 'hopr:test:hopr-ethereum'
+const log = debug(namespace)
```

Comment:
Just a note, you are mixing this separation of namespace and log with not having a separate namespace variable.

Reply:
Ah, that's correct. For now I'll stick with this and clean up all the other `namespace` variables used in the other test files.